apps/theming-designer/src/components/Samples.tsx

```diff
@@ -0,0 +1,132 @@
+            <Link disabled={true} href="http://dev.office.com/fabric/components/link">
+              Disabled link
+            </Link>
+            <SamplesCardHeader label="BUTTONS" />
+            <Stack horizontal>
+              <PrimaryButton text="Primary" />
+              <DefaultButton text="Standard" />
+            </Stack>
+            <SamplesCardHeader label="ACTION BUTTONS" />
+            <Stack horizontal gap={5}>
+              <ActionButton
+                data-automation-id="test"
+                iconProps={{ iconName: 'Add' }}
+                allowDisabledFocus={true}
+                disabled={false}
+                checked={true}
+              >
+                Action text
+              </ActionButton>
+              <ActionButton
+                data-automation-id="test"
+                iconProps={{ iconName: 'ChevronLeft' }}
+                allowDisabledFocus={true}
+                disabled={false}
+                checked={true}
+              />
+              <ActionButton
+                data-automation-id="test"
+                iconProps={{ iconName: 'Cancel' }}
+                allowDisabledFocus={true}
+                disabled={false}
+                checked={true}
+              />
+            </Stack>
+          </Stack>
+
+          <Stack gap={10}>
+            <SamplesCardHeader label="TEXT FIELD" />
+            <TextField label="Label" />
+            <TextField label="Label" disabled={true} />
+            <SamplesCardHeader label="TEXT FIELD - UNDERLINE" />
+            <TextField label="Label: " underlined />
+            <SamplesCardHeader label="DROPDOWN MENU" />
+            <Dropdown
+              placeholder="Content"
+              label="Label"
+              ariaLabel="Content dropdown"
+              options={[{ key: 'content', text: 'Content' }, { key: 'morecontent', text: 'More content' }]}
+            />
+          </Stack>
+
+          <Stack gap={10}>
+            <SamplesCardHeader label="TOGGLE" />
+            <Toggle
+              defaultChecked={true}
+              label="Enabled and checked"
+              onText="On"
+              offText="Off"
+              // onFocus={() => console.log('onFocus called')}
+              // onBlur={() => console.log('onBlur called')}
+              onChange={this._onToggleChange}
+            />
+            <SamplesCardHeader label="CHECKBOX" />
+            <Checkbox label="Standard checkbox" onChange={this._onCheckboxChange} />
+            <SamplesCardHeader label="RADIO BUTTON" />
+          </Stack>
+        </Stack>
+      </Card>
+    );
+  }
+  private _onToggleChange(ev: React.MouseEvent<HTMLElement>, checked: boolean | undefined) {
+    console.log('toggle is ' + (checked ? 'checked' : 'not checked'));
+  }
+  private _onCheckboxChange(ev?: React.FormEvent<HTMLElement | HTMLInputElement> | undefined, checked?: boolean | undefined) {
```

Comment:
Do we need these functions? They don't seem to be adding to much to the theme designer.

Reply:
It's a placeholder for the final version of the Samples when the checkbox might do more things. (the designers and I haven't reached a final design yet).